src/components/Page/index.tsx

```diff
@@ -52,13 +52,15 @@ const Page: React.FC<OwnProps> = ({
         <title>{`${title} | PINUS`}</title>
       </Head>
       <Navbar pathname={router.pathname} />
-      <Banner
-        title={title}
-        description={description}
-        bgImage={bgImage}
-        subBanner={subBanner}
-        renderSubcontent={renderSubcontent}
-      />
+      {title != "Aksara" ? (
```

Comment:
Instead of doing this, can add a new prop to let the component know whether to render `Banner` or not

Again, this is to keep the component as generic as possible

Reply:
done